### PR TITLE
doc: Remove outdated qt4 install information from README.md

### PR DESCRIPTION
**Pull request by laanwj**
This text is aimed at users installing the binaries.

Now that Qt5 is linked statically, there is no need to install Qt as a run-time dependency.

Needs backport to 0.12.x.

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -11,17 +11,11 @@ The following are some helpful notes on how to run Bitcoin on your native platfo
 
 ### Unix
 
-You need the Qt4 run-time libraries to run Bitcoin-Qt. On Debian or Ubuntu:
-
-	sudo apt-get install libqtgui4
-
 Unpack the files into a directory and run:
 
 - bin/32/bitcoin-qt (GUI, 32-bit) or bin/32/bitcoind (headless, 32-bit)
```
Comment (MarcoFalke) [inline doc/README.md]: There is no `bin/32`?

There are different tars for each arch?

Comment (laanwj) [inline doc/README.md]: Yes, good catch, that needs updating too.

**Comment by fanquake** [comment]
utACK once nits are fixed

**Comment by laanwj** [comment]
Removed the 32/64 bit reference.

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/8048/commits/6075bc4d67bcd5f95561dc6d53b4fdbaf2c63f4b

**Pull request merged**